sirepo/job_api.py

```diff
@@ -29,6 +31,21 @@
 _YEAR = datetime.timedelta(365)
 
 
+@api_perm.require_user
+def api_downloadDataFile(simulation_type, simulation_id, model, frame, suffix=None):
+#TODO(robnagler) validate suffix and frame
+    sim = sirepo.http_request.parse_params(
+        id=simulation_id,
+        model=model,
+        type=simulation_type,
+    )
+    f = _request(data=PKDict(sim.req_data, frame=frame, computeJobHash='x')).file
```

Comment:
Need to check the response to be "ok", and only then, read file. Sometimes data files aren't there.